### PR TITLE
Ensure Apache is started should be optional

### PR DESCRIPTION
**Pull request by pgilad**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -33,3 +33,6 @@ apache_mods_enabled:
   - rewrite.load
   - ssl.load
 apache_mods_disabled: []
+
+# Should apache service be started on task end
+apache_start_service: true
```
Comment (geerlingguy) [inline defaults/main.yml]: Let's call this `apache_state` with recommended values of `started` or `stopped`.

File: tasks/main.yml [hunk 2]
```diff
@@ -39,8 +39,13 @@
 - include: configure-Debian.yml
   when: ansible_os_family == 'Debian'
 
-- name: Ensure Apache is started and enabled on boot.
+- name: Ensure Apache is enabled on boot.
   service:
     name: "{{ apache_daemon }}"
-    state: started
```
Comment (geerlingguy) [inline tasks/main.yml]: Using the variable as defined per comment above, this line can remain as `state: {{ apache_state }}`.

File: tasks/main.yml [hunk 3]
```diff
@@ -39,8 +39,13 @@
...
     enabled: yes
+
+- name: Ensure Apache is started.
```
Comment (geerlingguy) [inline tasks/main.yml]: ...and we won't need an extra task.

File: tasks/main.yml (hunk 2 above)
Comment (pgilad) [inline tasks/main.yml]: ok great idea... thank.

**Comment by pgilad** [comment]
Rebased with your review notes

**Comment by pgilad** [comment]
As in https://github.com/geerlingguy/ansible-role-apache/issues/45#issuecomment-143348835 - the handler calls of `restart apache` need to be changed as well. They are called in configure step for both debian and red hat.
@geerlingguy Any suggestion what to to with these?

**Comment by geerlingguy** [comment]
Merging this now; we can think about the fix for the `restart apache` handler later, as this fix will at least resolve things like #31, whereas the restart handler should only cause some minor inconveniences and runs at the end of a playbook run.

**Pull request merged**